spring-web/src/main/java/org/springframework/http/converter/ResourceRegionHttpMessageConverter.java

```diff
@@ -201,10 +201,10 @@ private void writeResourceRegionCollection(Collection<ResourceRegion> resourceRe
 				print(out, "--" + boundaryString);
 				println(out);
 				if (contentType != null) {
-					print(out, "Content-Type: " + contentType.toString());
+					print(out, "Content-Type: " + contentType);
 					println(out);
 				}
-				Long resourceLength = region.getResource().contentLength();
+				long resourceLength = region.getResource().contentLength();
```

Comment:
Good catch.

That one was obviously just an accident.

Reply:
No more commits into CGLIB )